src/Raven.Client/ServerWide/Operations/WaitForDatabaseRecordDeletionOperation.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json.Converters;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
```

Comment:
Wrong approach, instead of doing that on the client, when the server side we delete a database, _there_ we'll wait